### ISSUE TITLE
Focus ring fails colour contrast

### ISSUE THREAD

**Issue by derekjohnson** [barnardos/consent-banner#3]
**Need**

_As a user of assistive technology
I want site content and interactions to be clear
So that I can use the website without unnecessary obstacles_

**Description**

The blue focus highlight for the buttons and link has insufficient contrast of 2.4:1 against the grey background and 1.1:1 against the green button background.

**Acceptance criteria**
- [ ] the focus ring around the link and buttons in the consent banner passes colour contrast tests

Related: https://github.com/barnardos/website/issues/1725
